src/main.rs

```diff
@@ -308,6 +308,19 @@ fn main() -> BError {
                 Effectors::DoubleDamage,
             ],
         ),
+        SkillDefinition::new(
+            Skills::DoubleAttackSpeed,
+            String::from("Double Attack Speed"),
+            String::from("double_attack_speed"),
+            String::from("Double the attack speed."),
+            0.0,
+            true,
```

Comment:
Is it intended that this is a passive skill?

Reply:
No, It's a misunderstanding of the 'passive' flag. 
If it's something that is triggered by the consumption of an item, I guess it's not passive then.